src/instructlab/configuration.py

```diff
@@ -127,6 +127,49 @@ def after_debug_level(self):
         return self
 
 
+class _rag_retriever_embedder(BaseModel):
+    """Class describing configuration of embedding parameters for RAG."""
+    model_name: str = Field(
```

Comment:
This is just the model name. There is no way to specify the models root folder, as with the `models_dir` field of the equivalent `EmbeddingModelConfig`.
Either change it to be a (absolute) path, or add a new field. First option seems to be the regular pattern adopted in ilab.